pkg/delete/delete.go

```diff
@@ -18,16 +18,17 @@ package delete
 
 import (
 	"fmt"
+	"github.com/dynatrace/dynatrace-configuration-as-code/internal/idutils"
 	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/client/dtclient"
 
-	"github.com/dynatrace/dynatrace-configuration-as-code/internal/idutils"
 	"github.com/dynatrace/dynatrace-configuration-as-code/internal/log"
 	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/api"
 )
 
 type DeletePointer struct {
```

Comment:
```suggestion
type DeletePointer coordinate.Coordinate
```
this is now just a coordinate righ?

Reply:
You're right! It is. And used as a coordinate as well

Reply:
On further thought, it actually isn't! For a classic Config API it is still defined by the Name, not the ConfigID. 
So while it looks similar and for Settings and Workflows it is, it's not always a coordinate. 

That's actually also why I had renamed it's `Identifier` field from the previously used `ConfigID`.

With that I actually think it's better to keep the separation and the util method to turn it into a coordinate in the specific places in which it is one

Reply:
Could you please add this to the docs? 
And add docs altogether, please? :)